yocto-base/00_flash_and_verify.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Library             Collections
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../../sonoff-rest-api/sonoff-api.robot
+Resource            ../../rtectrl-rest-api/rtectrl.robot
+Resource            ../../variables.robot
+Resource            ../../keywords.robot
+Resource            ../lib/sd-wire.robot
+
+Suite Setup         Prepare Test Suite
+Suite Teardown      Log Out And Close Connection
+
+
+*** Test Cases ***
+Verify number of connected SD Wire devices
```

Comment:
I'm not sure if this is a real test. It does nothing with the DUT itself, it simply checks our config file in variables.robot. Moreover, this is likely to be reused whenever SD Wire is in use.
It looks to me like a keyword to be put in SD wire library, called whenever we need to use SD Wire, or in some SD Wire init keyword. Perhaps as part of the Suite Setup as we do with RTE checking.

Reply:
@macpijan I adressed this in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/22/commits/483a919567615d797ef8a4f37e73c9e8b1449650). This commit:
* moves the `Verify Number Of Connected SD Wire Devices` to the `sd-wire.robot` resource
* changes a keyword in `keywords.robot`, to make it when the config is `rpi-3b` (the thing that uses the `sd-wire.robot` resource), the keyword `Verify Number Of Connected SD Wire Devices` is ran
* also adds a separate `IF ELSE` statement, for when the config is `rpi-3b`

Reply:
@macpijan also sorry, in the above comment i forgot to remove a section of code i had commented out. I did that in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/22/commits/f0df6a7cb46271996f139bb710e2e975321b800b)

Reply:
It is against guidelines to modify/add keywords in keywords.robot. But let's say we can ignore this rule this time.